src/backend/core/src/main/scala/com/lightbend/statefulserverless/Serve.scala

```diff
@@ -179,3 +176,114 @@ object Serve {
     }
   }
 }
+
+private[statefulserverless] object Names {
+  final def splitPrev(name: String): (String, String) = {
+    val dot = name.lastIndexOf('.')
+    if (dot >= 0) {
+      (name.substring(0, dot), name.substring(dot + 1))
+    } else {
+      ("", name)
+    }
+  }
+
+  final def splitNext(name: String): (String, String) = {
+    val dot = name.indexOf('.')
+    if (dot >= 0) {
+      (name.substring(0, dot), name.substring(dot + 1))
+    } else {
+      (name, "")
+    }
+  }
+}
+
+object Reflection {
+  import _root_.grpc.reflection.v1alpha._
+
+  private final val ReflectionPath = Path / ServerReflection.name / "ServerReflectionInfo"
+
+  def serve(fileDesc: FileDescriptor)(implicit mat: Materializer, sys: ActorSystem): PartialFunction[HttpRequest, Future[HttpResponse]] = {
+    implicit val ec: ExecutionContext = mat.executionContext
+    import ServerReflection.Serializers._
+
+    val handler = handle(fileDesc)
+
+    {
+      case req: HttpRequest if req.uri.path == ReflectionPath =>
+        val responseCodec = Codecs.negotiate(req)
+        GrpcMarshalling.unmarshalStream(req)(ServerReflectionRequestSerializer, mat)
+        .map( _.alsoTo(Sink.foreach(x => println(x.toString))).via(handler))
```

Comment:
println

Reply:
Doh! Good spotted, that's naturally there for me trying to reverse-engineer the spec.